api-publishing/src/main/java/org/zalando/nakadi/EventPublishingController.java

```diff
@@ -93,12 +97,15 @@ public ResponseEntity postBinaryEvents(@PathVariable final String eventTypeName,
             throws AccessDeniedException, BlockedException, ServiceTemporarilyUnavailableException,
             InternalNakadiException, EventTypeTimeoutException, NoSuchEventTypeException {
 
-        return status(HttpStatus.NOT_IMPLEMENTED).body("the method is under development");
-//        try {
-//            return postBinaryEvents(eventTypeName, batch, request, client, false);
-//        } catch (IOException e) {
-//            throw new InternalNakadiException("failed to parse batch", e);
-//        }
+        if ("acceptanceTest".equals(profile)) {
+            try {
+                return postBinaryEvents(eventTypeName, batch, request, client, false);
+            } catch (IOException e) {
+                throw new InternalNakadiException("failed to parse batch", e);
+            }
+        } else {
+            return status(HttpStatus.NOT_IMPLEMENTED).body("the method is under development profile = " + profile);
+        }
```

Comment:
Maybe let's wait for implementation to be enabled instead of relying on the test profile?

Reply:
I'll disable it for now.

Reply:
Disabled.